gtda/utils/validation.py

```diff
@@ -295,10 +297,11 @@ def check_point_clouds(X, distance_matrices=False, **kwargs):
         if not distance_matrices:
             if reduce(and_, (x.shape[0] == x.shape[1] for x in X), True):
                 warn(
-                    "All arrays are square. This is consistent with a "
-                    "collection of distance/adjacency matrices, but the input "
-                    "is being treated as a collection of vectors in Euclidean "
-                    "space.", DataDimensionalityWarning, stacklevel=2)
+                    "All arrays/matrices are square. This is consistent with "
+                    "a collection of distance/adjacency matrices, but the "
+                    "entries are being treated as collections of vectors in "
+                    "Euclidean space.", DataDimensionalityWarning,
+                    stacklevel=2)
 
             ref_dim = X[0].shape[1]  # Embedding dimension of first sample
             if not reduce(and_, (x.shape[1] == ref_dim for x in X[1:]), True):
```

Comment:
I don't think this warning should always be printed. It is perfectly allowed for the point clouds to have different embedded dimension. In practical use of the software this could generate a large number of lines in a log file that have no reason to be.

Reply:
I agree with this and I'll remove the warning!